### PR TITLE
Simulation docs

### PR DESCRIPTION
**Pull request by aauss**
Add docs for simulation module and change copyright year to make docs up to date

### CODE REVIEW

File: docs/3_data_simulation.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+Endemic Timeseries
+------------------
+An endemic timeseries is an timeseries of case counts (as defined in :ref:`outbreak-detection-formalization`.) that occurs naturally and varies between diseases, time, space, sex, age, and other dimensions. The important distinction is that there is no influence in the observed case counts due to an outbreak event.
+
+A simulation is usually a time-dependent, linear model that produces realistic case counts and can be set to mimic different types of disease dynamics. To achieve realism in epidemiological simulations, you would usually incorporate different effects into that model such as seasonality and trend e.g., a repetition of a certain pattern after one year with increasing case numbers over time. Finally, some distribution is used to make the outcome of the model non-deterministic. Since case counts are whole numbers, a Poisson or negative binomial distribution is used on top of the linear model to introduce some randomness in the observed case counts. These kinds of algorithms are referred to as ``seasonal_noise``.
```
Comment (JarnoRFB) [inline docs/3_data_simulation.rst:9]:
```suggestion
A simulation is usually a time-dependent, linear model that produces realistic case counts and can be set to mimic different types of disease dynamics. To achieve realism in epidemiological simulations, you would usually incorporate different effects into that model such as seasonality and trend e.g., a repetition of a certain pattern after one year with increasing case numbers over time. Finally, some distribution is used to make the outcome of the model non-deterministic. Since case counts are positive integers, a Poisson or negative binomial distribution is used on top of the linear model to introduce some randomness in the observed case counts. These kinds of algorithms are referred to as ``seasonal_noise``. 
```

File: docs/3_data_simulation.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Epidemic Timeseries
+-------------------
+Once we have created a model to simulate an endemic timeseries, we can introduce outbreak events by randomly increasing the case count at certain timepoints :math:`t`. A common approach is to have a chain of switching states (usually produces by a Markov chain) that use senseful transition probabilities to move into or leave the state of an outbreak.  Alternatively, one can assign certain timepoints to be in an outbreak to make such timeseries more comparable or tests edge cases of outbreak detection algorithms.
```
Comment (JarnoRFB) [inline docs/3_data_simulation.rst:13]:
```suggestion
Once we have created a model to simulate an endemic timeseries, we can introduce outbreak events by randomly increasing the case count at certain timepoints :math:`t`. A common approach is to have a chain of switching states (usually produces by a Markov chain) that use sensible transition probabilities to move into or leave the state of an outbreak.  Alternatively, one can assign certain timepoints to be in an outbreak to make such timeseries more comparable or tests edge cases of outbreak detection algorithms.
```

File: docs/conf.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 # -- Project information -----------------------------------------------------
 
 project = "epysurv"
-copyright = "2019, Rüdiger Busche"
+copyright = "2020, Rüdiger Busche"
```
Comment (JarnoRFB) [inline docs/conf.py:10]: I think you should add yourself to the authors ;)

File: docs/index.rst [hunk 4]
```diff
@@ -20,7 +20,8 @@ This package was originally developed at the Robert Koch Institute in the `Signa
 
    1_quickstart.rst
    2_outbreak_detection.rst
-   3_user_guide.rst
+   3_data_simulation.rst
```
Comment (JarnoRFB) [inline docs/index.rst:23]: I think there could be a better name for this, as we are not simulating the data, but the "epidemic", "outbreak", ... . Maybe "Simulating epidemiological dynamics" or just "Simulation"? Is there maybe an accepted term for this in the community?

File: docs/3_data_simulation.rst [hunk 5]
```diff
@@ -0,0 +1,15 @@
+Data Simulations
+==================
+Usually, public-health-related data can not be disclosed publicly which makes the development and systematic comparison of outbreak detection algorithms hard. Luckily, using simulations of hypothetical disease spreads is a valid alternative to real data. Thus, epysurv includes a module that allows you to simulate simple, univariate endemic and epidemic timeseries, i.e., with and without outbreak.
```
Comment (JarnoRFB) [inline docs/3_data_simulation.rst:3]: Is there maybe a review paper that backs some of the claims made here that we could reference, as in the outbreak detection section?

**Review by JarnoRFB — COMMENTED** [review]
Sorry for neglecting this for so long. Please address the comments that make sense to you, otherwise just go ahead and merge :+1:
